src/components/Button/ButtonGroup.js

```diff
@@ -0,0 +1,2 @@
+import { ButtonGroup } from 'react-bootstrap';
+export default ButtonGroup;
```

Comment:
actually - this can now be simply:
`export { ButtonGroup as default } from 'react-bootstrap';`

Reply:
would you mind sending me a small patch (using `style(buttongroup): change export style`) commit header?

Reply:
let me know if there is any issue @dabeng